SERVER_SIDE.md

```diff
@@ -71,6 +71,15 @@ var TomcatCookies = Cookies.withConverter({
 
 Alternatively, you can check the [Java Cookie](https://github.com/js-cookie/java-cookie) project, which integrates nicely with JavaScript Cookie.
 
+## Tomcat 8
+
+Since Tomat 8.0.15, there is a simple way to [say tomcat to be compliant with the RFC 6265](https://tomcat.apache.org/tomcat-8.0-doc/config/cookie-processor.html). It comes down to change your conf/context.xml file, adding the new CookieProcessor element nested inside the Context element. It would be like this:
```

Comment:
Sure, I agree the link is unnecessary. What about my suggestion of how it is structured, does it explain better?

"... Since Tomcat 8.0.15, it is possible to configure RFC 6265 compliance by changing your conf/context.xml file and adding the new \[CookieProcessor](https://tomcat.apache.org/tomcat-8.0-doc/config/cookie-processor.html) element ..."